.github/ISSUE_TEMPLATE.md

```diff
@@ -1,20 +1,27 @@
+<!--
 Thanks for taking the time for reporting an issue!
 
 Before reporting an issue on dd-trace-py, please be sure to provide all
 necessary information.
 
 If you're hitting a bug, make sure that you're using the latest version of this
 library.
+-->
+
+### Summary of problem
 
 ### Which version of dd-trace-py are you using?
 
 ### Which version of pip are you using?
 
 _ddtrace requires pip>=18 to install one of our pre-built wheels_
```

Comment:
we can probably get rid of this now? Given `pip==18` is already 4 years old, and I don't see too many issues with this these days?